contrib/resources/glshaders/misc/fixvideo.glsl

```diff
@@ -0,0 +1,171 @@
+#version 120
+
+/*
+ *  Adaptation of the old fixvideo.fx Daum shader originally written by Ove
+ *  Kavven with some improvements and an intelligent scanline-doubling method
+ *  added by John Novak.
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2023 The DOSBox Team
+ *  Copyright (C) 2015 Ove Kaaven
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+// Deinterlace method
+// ------------------
+//   0.0 = Mode 0: vertical average + brightness-boost
+//   1.0 = Mode 1: intelligent scanline-doubling
+#define DEINTERLACE_METHOD 1.0
+
+// Mode 0 params
+// -------------
+//   Brightness boost (0.00 to 1.00)
+#define BRIGHT_BOOST 0.50
+
+// Mode 1 params
+// -------------
+//   4 sample points to check if the scanline above the current is black;
+//   might need to tweak it a bit per game
+//   0.00 = leftmost pixel; 0.50 = middle pixel; 1.0 = rightmost pixel
+#define SAMPLE_POINT_1 0.00
+#define SAMPLE_POINT_2 0.25
+#define SAMPLE_POINT_3 0.75
+#define SAMPLE_POINT_4 1.00
+
+//   Intensity of the deinterlaced odd & even scanlines.
+//   Use values above 1.0 for extra brightness boost.
+//   1.00 = full intensity, 0.0 = zero intensity (black)
+#define ODD_LINE_INTENSITY  1.00
+#define EVEN_LINE_INTENSITY 1.00
```

Comment:
The only difference between the two shaders is the `ODD_LINE_INTENSITY` and `EVEN_LINE_INTENSITY` settings.